client/src/app/components/auth-tabs/auth-tabs.component.ts

```diff
@@ -0,0 +1,13 @@
+import { Component, OnInit } from '@angular/core';
+import { FormControl, Validators } from '@angular/forms';
+
+@Component({
+  selector: 'app-auth-tabs',
+  templateUrl: './auth-tabs.component.html',
+  styleUrls: ['./auth-tabs.component.css'],
+})
+export class AuthTabsComponent implements OnInit {
+  constructor() {}
+
+  ngOnInit(): void {}
+}
```

Comment:
What is the purpose of this component?

Are we going to add functionality later?

Reply:
If you look at the auth-tabs.component.html it acts as a tab to switch between login and register component as shown in the screenshot below:
<img width="662" alt="Screen Shot 2020-10-05 at 11 20 36 PM" src="https://user-images.githubusercontent.com/35696935/95155095-62752a80-0761-11eb-8091-f236afcd39d4.png">